Lib/unittest/case.py

```diff
@@ -911,6 +919,15 @@ def assertAlmostEqual(self, first, second, places=None, msg=None,
                 safe_repr(second),
                 safe_repr(delta),
                 safe_repr(diff))
+        elif rel_delta is not None:
+            if diff < rel_delta*abs(first):
```

Comment:
```suggestion
            if isclose(first,second,rel_tol=rel_delta, abs_tol=0.0):
```

`import math` is necessary at the beginning of the file if you want to inlcude this suggestion